### DISCUSSION TITLE
How to show matplotlib figure in the android app?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1296] [Q&A]
Hi,  I want to show matplotlib figure in the android app, so I can  enlarge the image and click on a point to display the value. How can I show matplotlib figure in the android app? 

The matplotlib figure is like this,
![image](https://github.com/beeware/briefcase/assets/134466342/8352d031-8dbc-4dd8-ac88-ccf323f49f05)

**Comment by mhsmith** [comment]
There's an example of how to show a matplotlib figure in an Android app at https://github.com/beeware/briefcase-android-gradle-template/pull/52. Unfortunately, there's currently no easy way for it to react to clicks on the figure itself. You could try adding some buttons outside of the figure instead.
